DOCS.md

```diff
@@ -9,84 +9,129 @@ image: lucap/drone-now
 ---
 
 The Now plugin deploy your build to [now.sh](https://zeit.co/now). The below pipeline configuration demonstrates simple usage:
+Note: `deploy_name` is an optional parameter. If it is not given now.sh will deploy the root directory of your application.
 
 ```yaml
 pipeline:
   now:
     image: lucap/drone-now
-    token: xxxxx
-    name: deployment-name
+    deploy_name: deployment-name
+    secrets: [ now_token ]
 ```
 
-Example configuration for assigning [Aliases and Domains](https://zeit.co/docs/features/aliases):
+Set the directory you want to deploy. See the [Deployment Documentation](https://zeit.co/docs/getting-started/deployment#) for more info.
+
+```yaml
+pipeline:
+  now:
+    image: lucap/drone-now
+    deploy_name: deployment-name
+    secrets: [ now_token ]
++   directory: public
+```
+
+Example configuration with team scope:
 
 ```diff
 pipeline:
   now:
     image: lucap/drone-now
-    token: xxxxx
--   name: deployment-name
-+   alias: my-deployment-alias
+    deploy_name: deployment-name
+    secrets: [ now_token ]
+    directory: public
++   team: myteam
 ```
 
-Example configuration with custom [Path Aliases](https://zeit.co/docs/features/path-aliases):
+
+Example configuration to enforce type (by default, it's detected automatically):
 
 ```diff
 pipeline:
   now:
     image: lucap/drone-now
-    token: xxxxx
--   alias: my-deployment-alias
-+   rules_domain: example.com
-+   rules_file: rules.json
+    deploy_name: deployment-name
+    secrets: [ now_token ]
+-   directory: public
+-   team: myteam
++   type: npm
 ```
 
-Example configuration for [Scaling](https://zeit.co/docs/getting-started/scaling):
+Example configuration for assigning [Aliases and Domains](https://zeit.co/docs/features/aliases):
 
 ```diff
 pipeline:
   now:
     image: lucap/drone-now
-    token: xxxxx
--   rules_domain: example.com
--   rules_file: rules.json
-+   scale: 2
+    deploy_name: deployment-name
+    secrets: [ now_token ]
+-   type: npm
++   alias: my-deployment-alias
 ```
 
-Example configuration to enforce type (by default, it's detected automatically):
+Example configuration for [Cleaning Up Old Deployments](https://zeit.co/docs/features/aliases):
```

Comment:
Cleaning up doesn't come from this documentation.

Reply:
fixed!